frontend/src/employee-frontend/components/unit/unit-reservations/ChildDay.tsx

```diff
@@ -79,19 +74,25 @@ export default React.memo(function ChildDay({ day, childReservations }: Props) {
           <ReservationTime>Vapaapäivä</ReservationTime>
         ) : (
           /* else show no reservation */
-          <ReservationTime>Ei varausta</ReservationTime>
+          <ReservationTime warning>Sop.aika puuttuu</ReservationTime>
```

Comment:
Should these missing translation be fixed now?

Reply:
Fixed now.